requirements.txt

```diff
@@ -3,17 +3,14 @@ scipy>=1.11.4
 amazon-braket-sdk==1.79.1
 aws-configure==2.1.8
 amazon-braket-default-simulator==1.23.2
-boto3>=1.34.107
+boto3==1.34.124
```

Comment:
I don't like hardcoding the dependancies if not needed

Reply:
If I remember well it was generating a conflict with the dependencies of the new qiskit packages, so I had to freeze it somehow, but maybe we can find a better solution, or double check this fact.

Reply:
we can have a task just for this. Indeed, it's not only about this package, but about all (or close to) packages